memecity.engine/RawFontWrapper.cpp

```diff
@@ -0,0 +1,11 @@
+﻿#include "RawFontWrapper.h"
+
+RawFontWrapper::RawFontWrapper(TTF_Font* font)
```

Comment:
list initialization kan hier toegepast worden? en omdat de wrappers zo klein zijn, zouden ze in principe allemaal in de header file thuispassen.